src/components/DeploymentsTable.vue

```diff
@@ -2,31 +2,23 @@
   <div class="deployments-table">
     <p-layout-table sticky :root-margin="margin">
       <template #header-start>
-        <div class="deployments-table__header-start">
-          <ResultsCount v-if="selectedDeployments.length == 0" label="Deployment" :count="deploymentsCount" />
-          <SelectedCount v-else :count="selectedDeployments.length" />
+        <ResultsCount v-if="selectedDeployments.length == 0" label="Deployment" :count="total" />
+        <SelectedCount v-else :count="selectedDeployments.length" />
 
-          <DeploymentsDeleteButton v-if="can.delete.deployment" :selected="selectedDeployments" @delete="deleteDeployments" />
-        </div>
+        <DeploymentsDeleteButton v-if="can.delete.deployment" small :selected="selectedDeployments" @delete="deleteDeployments" />
       </template>
 
       <template #header-end>
-        <div class="deployments-table__header-end">
```

Comment:
It looks like at smaller screens this used to wrap this section rather than overflow?

| Before | After |
| ---- | ---- |
| <img width="315" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/8c17747d-2241-4f58-b1f8-73e03cff416a"> | <img width="317" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/69c466e8-2e61-4741-b782-9e08faa8e057"> |

Reply:
Ah it's also losing a tiny bit of margin between the header start/end (because the margin was on the end class)

<img width="313" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/a162694d-300e-4659-8b89-60459fc0867d">


vs

<img width="314" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/cbbb6a3c-1f19-4b42-bacf-cab044669f34">

Reply:
Ah good catch. Meant to fix this in the table layout component. Will address

Reply:
it feels _maybe_ like PLayoutTable's header should have a little flex gap on it?

Reply:
Agreed

Reply:
PR for that here https://github.com/PrefectHQ/prefect-design/pull/972

Reply:
Bumped prefect-design to latest to bring in that change 👍

Reply:
cool. i went and looked at some other p-layout-tables (just in nebula) and looks like only these tables had margin explicitly in the children so elsewhere, we should just be _gaining_ some nice space at denser heads